docs/pipelines.md

```diff
@@ -227,6 +228,57 @@ spec:
           value: /workspace/examples/microservices/leeroy-web
 ```
 
+### Tekton Bundles
+
+You may also specify your `Task`
+reference using a `Tekton Bundle`. A `Tekton Bundle` is an OCI artifact that
+contains Tekton resources like `Tasks` and `Pipelines` which can be referenced
```

Comment:
NIT:  this section holds a list of `PipelineTasks`, OCI reference can not refer to a `pipeline` here. It sounds confusing to include `pipelines` here.